<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PGK - Zadanie #15: Oświetlenie + cieniowanie 3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>270</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>Rzut perspektywiczny</string>
    </property>
    <widget class="QSlider" name="distanceSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>175</number>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>d</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="translationGroupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>270</width>
      <height>105</height>
     </rect>
    </property>
    <property name="title">
     <string>Przesunięcie</string>
    </property>
    <widget class="QSlider" name="translationXSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-300</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>55</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QSlider" name="translationYSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-300</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationZSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rotationGroupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>270</width>
      <height>105</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrót</string>
    </property>
    <widget class="QSlider" name="rotationXSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>55</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QSlider" name="rotationYSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotationZSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scalingGroupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>270</width>
      <height>105</height>
     </rect>
    </property>
    <property name="title">
     <string>Skalowanie</string>
    </property>
    <widget class="QSlider" name="scalingXSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>55</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QSlider" name="scalingYSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingZSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="optionGroupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>270</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Opcje</string>
    </property>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Resetuj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Zakończ</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>425</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Źródło światła 1</string>
     </attribute>
     <widget class="QSlider" name="light1YSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-500</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSlider" name="light1XSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>5</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-500</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="light1ZSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>55</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-500</number>
      </property>
      <property name="maximum">
       <number>-200</number>
      </property>
      <property name="value">
       <number>-200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>55</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Źródło światła 2</string>
     </attribute>
     <widget class="QSlider" name="light2XSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>5</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSlider" name="light2ZSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>55</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>55</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QSlider" name="light2YSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>405</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pozycje źródeł światła</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
